styles/stylesheet.css

```diff
@@ -0,0 +1,380 @@
+  color: #213d69;
+  font-size: 1.5rem;
+  margin: 2rem 0 0.4rem 0; /*check*/
+}
+h6 {
+  font-size: 0;
+  position: absolute;
+  left: 1000rem;
+}
+.occupy-solo {
+  width: 100%;
+}
+.hidden {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+}
+.hidden:focus {
+  left: 0;
+}
+.top-nav {
+  background-color: #d7dbd8;
+  font-weight: bold;
+  color: #213d69;
+  font-size: 1.2rem;
+  display: flex;
+}
+.intro-text {
+  font-size: 1.4rem;
+  color: #5d6f73;
+  font-weight: lighter;
+}
+.link-to-blog {
+  color: #79b6f2;
+}
+/*Stuff that will likely go into an @media device*/
+.head-container {
+  width: 85%;
+}
+.cart-holder {
+  width: 10%;
+}
+.standard-width {
+  width: 85%;
+}
+.main {
+  width: 70%;
+  margin-top: 3rem;
+}
+.dog-grid {
+  grid-gap: 2rem 1rem;
+  margin-top: 1rem;
+}
+.footer-text {
+  width: 33%;
+  padding: 3rem 0 4rem 0;
+}
+.bottom-spacer {
+  margin-bottom: 5rem;
+}
+.top-spacer {
+  margin-top: 2rem !important;
+}
+.basic-input {
+  width: 70%;
+}
+.city-state-zip-holder {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 70%;
+}
+.city {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 40%;
+}
+.state-zip {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 25%;
+}
+.blog-img {
+    object-fit: cover;
+    object-position: center;
+    width: 25%;
+    height: 15rem;
+    margin-right: 1rem;
+}
+#pickup-location {
+  width: 40%;
+}
+#intro-image {
+    object-fit: cover;
+    height: 500px;
+    margin-bottom: 2rem;
+}
+/*End*/
+@media screen and (min-width: 992px) {
+  #grid-1 {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+  #grid-2 {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+  aside {
+    margin-top: 3rem;
+    width: 25%;
+  }
+  .divide {
+    display: flex;
+    justify-content: space-between;
+  }
+  .head-container {
+      padding-bottom: 9rem;
+      padding-top: 3rem;
+  }
+  .small-blog-image {
+      object-fit: cover;
+      height: 15rem;
+      object-position: center;
+  }
+}
+@media screen and (min-width: 768px) and (max-width: 992px) and (orientation: landscape) {
+  #grid-1 {
+    grid-template-columns: 1fr 1fr;
+  }
+  #grid-2 {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+  aside {
+    margin-top: 3rem;
+    width: 25%;
+  }
+  .divide {
+    display: flex;
+    justify-content: space-between;
+  }
+  .head-container {
+      padding-bottom: 9rem;
+      padding-top: 3rem;
+  }
+  .small-blog-image {
+      object-fit: cover;
+      height: 10rem;
+      object-position: center;
+  }
+}
+@media screen and (min-width: 768px) and (orientation: portrait) {
+  #grid-1 {
+    grid-template-columns: 1fr 1fr;
+  }
+  #grid-2 {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+  aside {
+    display: none;
+  }
+  .divide {
+    display: flex;
+    justify-content: center;
+  }
+  .head-container {
+      padding-bottom: 4rem;
+      padding-top: 2rem;
+  }
+}
```

Comment:
You have a dead zone between 768 and 600 px where the aside becomes visible again and throws off the layout.